### PR TITLE
darts: fix missing closing parenthesis

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/darts/description.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ In our particular instance of the game, the target rewards 4 different amounts o
 - If the dart lands in the inner circle of the target, player earns 10 points.
 
 The outer circle has a radius of 10 units (this is equivalent to the total radius for the entire target), the middle circle a radius of 5 units, and the inner circle a radius of 1.
-Of course, they are all centered at the same point (that is, the circles are [concentric][] defined by the coordinates (0, 0).
+Of course, they are all centered at the same point (that is, the circles are [concentric][] defined by the coordinates (0, 0)).
```
Comment (ErikSchierboom) [inline exercises/darts/description.md:15]: One could argue that this makes it less readable. Let me know what you all think

**Review by kytrinyx — APPROVED** [review]
I think this makes sense. If we don't do this we should at least remove the opening one!

**Comment by glennj** [comment]
A semicolon works here, I think:
```
Of course, they are all centered at the same point; that is, the circles are [concentric][] defined by the coordinates (0, 0).
```
or a hyphen.

**Review by angelikatyborska — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

File: exercises/darts/description.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply exercises/darts/description.md]: Much better, yes!

**Pull request merged**